protocol/communities_messenger_test.go

```diff
@@ -3905,3 +3905,27 @@ func (s *MessengerCommunitiesSuite) TestRequestAndCancelCommunityAdminOffline()
 	s.Require().NotEmpty(cancelRequestToJoin2.Clock)
 	s.Require().Equal(cancelRequestToJoin2.PublicKey, common.PubkeyToHex(&s.alice.identity.PublicKey))
 }
+
+func (s *MessengerCommunitiesSuite) TestCommunityLastOpenedAt() {
+	community, _ := s.createCommunity()
+	s.advertiseCommunityTo(community, s.owner, s.alice)
+	s.joinCommunity(community, s.owner, s.alice)
+
+	// Mock frontend triggering communityUpdateLastOpenedAt
+	response, err := s.alice.CommunityUpdateLastOpenedAt(community.IDString())
+	s.Require().NoError(err)
+
+	// Check lastOpenedAt was updated
+	lastOpenedAt1 := response.Communities()[0].LastOpenedAt()
+	s.Require().True(response.Communities()[0].LastOpenedAt() > 0)
+
+	// Nap for a bit
+	time.Sleep(time.Second)
+
+	// Check lastOpenedAt was successfully updated twice
+	response, err = s.alice.CommunityUpdateLastOpenedAt(community.IDString())
+	lastOpenedAt2 := response.Communities()[0].LastOpenedAt()
+	s.Require().NoError(err)
+
+	s.Require().True(lastOpenedAt2 > lastOpenedAt1)
+}
```

Comment:
Can you please add a test that a sync message arrives to another instance?

Reply:
Done